### ISSUE TITLE
[0.9.50] input文本框的dropdown闪退

### ISSUE THREAD

**Issue by laphylee** [brookhong/Surfingkeys#1060]
## Version

SurfingKeys: 0.9.50

Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.70 Safari/537.36

## Issue

我在多个网站上发现了input文本框的dropdown闪退问题，例如outlook web, 滴答清单等。
以outlook web上添加一个会议为例，将输入定位到"search for a room or location"的input文本框时，本来会出现多个候选会议室的dropdown供选择。现在这个dropdown出现后立马闪退。

**正常情况** (V0.9.48)

![Surfingkeys_Before](https://user-images.githubusercontent.com/13502835/67773220-57684380-fa96-11e9-9215-a35b7550a1ae.gif)

**现在**  (V0.9.50)

![Surfingkeys_After](https://user-images.githubusercontent.com/13502835/67774063-b5e1f180-fa97-11e9-8b15-0af1a59a4995.gif)

**Comment by amaya382** [comment]
I encountered the same problem at `Slack > Add custom emoji > "Give it a name" form`

```
SurfingKeys: 0.9.50
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.121 Safari/537.36 Vivaldi/2.8.1664.44
```

**Comment by brookhong** [comment]
A fix has been released with 0.9.51, please check it out, and see if it works for you.

**Comment by amaya382** [comment]
Thank you @brookhong, I confirmed that it has been fixed with 0.9.51!

**Comment by cvb** [comment]
Having this problem with
firefox 70.0.1 (64-bit)
SurfingKeys 0.9.53
at sites with dropdowns like https://www.nyse.com/quote/XNGS:YNDX (buttons on the chart)

**Comment by brookhong** [comment]
@cvb 
The issue with site nyse has been fixed in 0.9.54, please have a try.

**Comment by cvb** [comment]
It works, thanks!
